doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1814 @@
+
+=item RSA_get0_engine
+
+See L</Providers are a replacement for engines and low-level method overrides>
+
+=item RSA_get0_crt_params, RSA_get0_d, RSA_get0_dmp1, RSA_get0_dmq1,
+RSA_get0_e, RSA_get0_factors, RSA_get0_iqmp, RSA_get0_key,
+RSA_get0_multi_prime_crt_params, RSA_get0_multi_prime_factors, RSA_get0_n,
+RSA_get0_p, RSA_get0_pss_params, RSA_get0_q and RSA_get_multi_prime_extra_count.
+
+See L</Deprecated low-level key parameter getters>
+
+=item RSA_new, RSA_free and RSA_up_ref
+
+See L</Deprecated low-level object creation>.
+
+=item RSA_get_default_method, RSA_get_ex_data and RSA_get_method
+
+See L</Providers are a replacement for engines and low-level method overrides>.
+
+=item RSA_get_version
+
+There is no replacement.
+
+=item RSA_meth_*, RSA_new_method, RSA_null_method and RSA_PKCS1_OpenSSL
+
+See L</Providers are a replacement for engines and low-level method overrides>.
+
+=item RSA_padding_add_* and RSA_padding_check_*
+
+See L</Deprecated low-level signing functions> and
+L</Deprecated low-level encryption functions>.
+
+=item RSA_print and RSA_print_fp
+
+See L</Deprecated low-level key printing functions>
+
+=item RSA_public_encrypt and RSA_private_decrypt
+
+See L</Deprecated low-level encryption functions>
+
+=item RSA_private_encrypt and RSA_public_decrypt
+
+This is equivalent to doing sign and verify operations (with a padding mode
+of none). See L</Deprecated low-level signing functions>.
+
+=item RSAPrivateKey_dup, RSAPublicKey_dup
+
+There is no direct replacement. Applications may use L<EVP_PKEY_dup(3)>.
+
+=item RSAPublicKey_it and RSAPrivateKey_it
+
+See L</Deprecated low-level key reading and writing functions>
+
+=item RSA_set0_crt_params, RSA_set0_factors, RSA_set0_key and RSA_set0_multi_prime_params
+
+See L</Deprecated low-level key parameter setters>.
+
+=item RSA_set_default_method, RSA_set_method, RSA_set_ex_data
+
+See L</Providers are a replacement for engines and low-level method overrides>
+
+=item RSA_sign, RSA_sign_ASN1_OCTET_STRING, RSA_verify, RSA_verify_ASN1_OCTET_STRING,
+RSA_verify_PKCS1_PSS and RSA_verify_PKCS1_PSS_mgf1
+
+See L</Deprecated low-level signing functions>.
+
+=item RSA_X931_derive_ex, RSA_X931_generate_key_ex and RSA_X931_hash_id.
+
+There are no replacements for these functions.
+X931 padding can be set using L<EVP_SIGNATURE-RSA(7)/Signature Parameters>.
+See B<OSSL_SIGNATURE_PARAM_PAD_MODE>.
+
+=item SEED_encrypt, SEED_decrypt, SEED_set_key, SEED_cbc_encrypt,
+SEED_cfb128_encrypt, SEED_ecb_encrypt and SEED_ofb128_encrypt.
+
+See L</Deprecated low-level encryption functions>.
+The SEED algorithm has been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item SHA1_Init, SHA1_Update, SHA1_Final and SHA1_Transform
+
+=item SHA224_Init, SHA224_Update and SHA224_Final
+
+=item SHA256_Init, SHA256_Update, SHA256_Final and SHA256_Transform
+
+=item SHA384_Init, SHA384_Update and SHA384_Final
+
+=item SHA512_Init, SHA512_Update, SHA512_Final and SHA512_Transform
```

Comment:
Again, five =item lists that should be merged.  Perhaps keep SHA1 separate and duplicate the text.